gmm_client/test/naive_queries_SUITE.erl

```diff
@@ -0,0 +1,165 @@
+    client:add_vertex(<<"zone2">>, <<"user">>, <<"User3">>),
+    client:add_vertex(<<"zone3">>, <<"user">>, <<"User4">>),
+    client:add_vertex(<<"zone3">>, <<"user">>, <<"User5">>),
+    client:add_vertex(<<"zone1">>, <<"group">>, <<"Group1">>),
+    client:add_vertex(<<"zone2">>, <<"group">>, <<"Group2">>),
+    client:add_vertex(<<"zone2">>, <<"group">>, <<"Group3">>),
+    client:add_vertex(<<"zone3">>, <<"group">>, <<"Group4">>),
+    client:add_vertex(<<"zone1">>, <<"space">>, <<"Space1">>),
+    client:add_vertex(<<"zone1">>, <<"space">>, <<"Space2">>),
+    client:add_vertex(<<"zone2">>, <<"space">>, <<"Space3">>),
+    client:add_vertex(<<"zone3">>, <<"space">>, <<"Space4">>),
+    client:add_vertex(<<"zone1">>, <<"provider">>, <<"Provider1">>),
+    client:add_vertex(<<"zone2">>, <<"provider">>, <<"Provider2">>),
+    client:add_vertex(<<"zone3">>, <<"provider">>, <<"Provider3">>),
+
+    client:add_edge(<<"zone1/User1">>, <<"zone1/Group1">>, <<"01110">>, <<"trace1">>),
+    client:add_edge(<<"zone1/Group1">>, <<"zone1/Provider1">>, <<"10101">>, <<"trace2">>),
+    client:add_edge(<<"zone1/User2">>, <<"zone3/Space4">>, <<"00100">>, <<"trace3">>),
+    client:add_edge(<<"zone2/User3">>, <<"zone2/Group3">>, <<"00011">>, <<"trace4">>),
+    client:add_edge(<<"zone2/Group3">>, <<"zone2/Group2">>, <<"00010">>, <<"trace5">>),
+    client:add_edge(<<"zone2/Group2">>, <<"zone1/Space2">>, <<"10000">>, <<"trace6">>),
+    client:add_edge(<<"zone2/Space3">>, <<"zone2/Provider2">>, <<"01100">>, <<"trace7">>),
+    client:add_edge(<<"zone3/User4">>, <<"zone2/Group3">>, <<"01000">>, <<"trace8">>),
+    client:add_edge(<<"zone2/Group3">>, <<"zone3/Group4">>, <<"10000">>, <<"trace9">>),
+    client:add_edge(<<"zone3/User5">>, <<"zone3/Group4">>, <<"01101">>, <<"trace10">>),
+    client:add_edge(<<"zone3/Group4">>, <<"zone3/Space4">>, <<"10111">>, <<"trace11">>),
+    client:add_edge(<<"zone3/Space4">>, <<"zone2/Provider2">>, <<"11110">>, <<"trace12">>),
+
+    % then
+    {ok, MembersResponse1} = client:naive_members(<<"zone3/Space4">>),
+    {ok, MembersResponse2} = client:naive_members(<<"zone2/Group2">>),
+    {ok, MembersResponse3} = client:naive_members(<<"zone1/Space1">>),
+    {ok, MembersResponse4} = client:naive_members(<<"zone1/Provider1">>),
+
+    {ok, ReachesResponse1} = client:naive_reaches(<<"zone2/User3">>, <<"zone3/Provider3">>),
+    {ok, ReachesResponse2} = client:naive_reaches(<<"zone3/User5">>, <<"zone3/Space4">>),
+    {ok, ReachesResponse3} = client:naive_reaches(<<"zone2/User3">>, <<"zone2/Provider2">>),
+    {ok, ReachesResponse4} = client:naive_reaches(<<"zone1/User1">>, <<"zone1/Provider1">>),
+
+    {ok, EPResponse1} = client:naive_effective_permissions(<<"zone1/User1">>, <<"zone1/Provider1">>),
+    {ok, EPResponse2} = client:naive_effective_permissions(<<"zone3/User5">>, <<"zone2/Provider2">>),
+    {ok, EPResponse3} = client:naive_effective_permissions(<<"zone2/User3">>, <<"zone2/Provider2">>),
```

Comment:
W zasadzie można to trochę zmienić. Tzn. duration i tak tutaj ignorujemy, więc proponuję zrobić to tak:

```suggestion
    {ok, #{<<"members">> := MembersResponse1}} = client:naive_members(<<"zone3/Space4">>),
    %% ...

    {ok, #{<<"reaches">> := ReachesResponse1}} = client:naive_reaches(<<"zone2/User3">>, <<"zone3/Provider3">>),
    %% ...

    {ok, #{<<"effectivePermissions">> := EPResponse1}} = client:naive_effective_permissions(<<"zone1/User1">>, <<"zone1/Provider1">>),
    %% ...
```